packages/vue/src/components/CcvChoroplethChart.ts

```diff
@@ -1,7 +1,4 @@
-import {
-	ExperimentalChoroplethChart as ChoroplethChart,
-	type ChoroplethChartOptions
-} from '@carbon/charts'
+import { ChoroplethChart, type ChoroplethChartOptions } from '@carbon/charts'
 import { chartFactory } from './chartFactory'
 
 export default chartFactory<ChoroplethChartOptions>(
```

Comment:
we should still export ExperimentalChoroplethChart as well and mark as deprecated

Reply:
The export is in packages/vue/src/index.ts. Both are exported with the experimental one marked deprecated.